lib/client/js/ui/controls/Menu.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Insert description of Menu class
+ *
+ * @private
+ * @class
+ * @returns {Menu}
+ */
+var Menu = (function () {
+
+    Menu.prototype = new Control();
+
+    /**
+     * Decorates a jquery object with the header
+     *
+     * @private
+     * @function viewHeader
+     */
+    var viewHeader = function(jqueryObj) {
+        return jqueryObj.addChild("header");
+    };
+
+    /**
+     * Decorates a jquery object with the connections list
+     *
+     * @private
+     * @function Menu#viewConnectionsList
```

Comment:
This isn't an instance method of Menu - it's just a private inner function. Feel free to omit the name of the function - JSDoc will pick it up.

```
@private
@function
@inner
```